traffic_ops/traffic_ops_golang/parameters.go

```diff
@@ -94,7 +96,11 @@ func getParameters(v url.Values, db *sqlx.DB, privLevel int) ([]tc.Parameter, er
 
 	query, queryValues := dbhelpers.BuildQuery(v, selectParametersQuery(), queryParamsToSQLCols)
 
-	query += ParametersGroupBy()
+	queryPieces := strings.Split(query, "ORDER BY")
```

Comment:
This makes me anxious, it doesn't handle quoting and escaping. What if the query has the words "order by" in a SQL string for some reason?

Reply:
The query isn't user specified so unless a developer adds a second ORDER BY to the query, which would fail anyway, this won't be a problem.

Reply:
ugh..   yep -- you're right...   thanks for catching that.

@DylanVolz let's talk about this tomorrow.